modules/ROOT/pages/about-my-applications.adoc

```diff
@@ -12,14 +12,30 @@ image::ex2-my-applications.png[]
 A contract holds information about the contract with a client ID and client secret for the API.
 Each application lists a description, its URL, redirect URIs, grant types, and a usage dashboard.
 
-The usage information lets you set the verification duration. If not specified, the range is the last 8 days. The usage lists the total number of requests to the API, average latency, the error rate, and graphs for total requests, average latency, requests by HTTP status code, and the error percentage.
-
-The applications result when users click Request Access to establish
-a link between an application and an API. Either the application already exists,
-or the user creates the application from the Request Access menu.
+The usage information enables you to set the verification duration. If not specified,
+the range is the previous eight days. The usage lists the total number of requests to
+the API, average latency, the error rate, and graphs for total requests,
+requests by HTTP status code, and the error percentage.
 
 Applications in My Applications are registered using API Manager in Anypoint Platform.
 
+To implement an API, the client application must first request access. When the
+request is approved by the API, a contract is created between the client
+application and the API, and the client application is registered.
+
+If you don't have an application to link to your API, you can create the application
+in the Request Access menu. You can then click Request Access to connect your
+application and the API.
```

Comment:
This is a basic case, that can be all right. For your radar: If at least one SLA is defined in the API, you have to choose one when requesting access. Also, terms and conditions have to be accepted, if defined.

Reply:
This is good info. Added.